samples/wifi/shell/boards/nrf54h20dk_nrf54h20_cpuapp_soc1.overlay

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2024 Nordic Semiconductor ASA
```

Reply:
Generally for cherry picks we don't modify anything, NCS has many older copyrights.